utils/constants.ts

```diff
@@ -41,3 +43,75 @@ export const CATEGORIES_PLACEHOLDERS: Record<Category, string> = {
 };
 
 export const KNOWN_LSP_ADDRESS = "0x372802d8A2D69bB43872a1AABe2bd403a0FafA1F";
+
+export const infuraId = process.env.NEXT_PUBLIC_INFURA_ID || "";
```

Comment:
not sure if we have a place to document this, but should always try to document new envs

Reply:
Definitely README. it should probably be noted in the PR even?